awx/main/migrations/0176_instance_group_admin_migration.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.16 on 2023-02-17 02:45
+
+from django.db import migrations
+from awx.main.migrations import _rbac as rbac
+from awx.main.migrations import _migration_utils as migration_utils
+from awx.main.migrations import _OrgAdmin_to_use_ig as oamigrate
+from awx.main.migrations import ActivityStreamDisabledMigration
+
+
+class Migration(ActivityStreamDisabledMigration):
+    dependencies = [
+        ('main', '0175_instance_group_role_addition'),
+    ]
+    operations = [
+        migrations.RunPython(migration_utils.set_current_apps_for_migrations),
+        migrations.RunPython(rbac.create_roles),
+        migrations.RunPython(oamigrate.migrate_org_admin_to_use),
```

Comment:
Why not just add the function into this file?

Reply:
Have you tried reversing this migration? If it can't do it on its own and its something we wanted to support, there is a way to tell Django how to reverse a migration see `0170_node_and_link_state.py` for details if you are interested in that.

Reply:
for this we don't really support backing out migrations so its probably okay.